elm/src/UiFramework/Padding.elm

```diff
@@ -0,0 +1,56 @@
+module UiFramework.Padding exposing (responsive)
+
+{-
+    Responsive padding using fillPortion
+-}
+
+import Element exposing (..)
+import SharedState exposing (SharedState)
+
+
+
+{-
+      Elm-ui does not have a "percent padding". For example, I can't get a 50% padding because I have to use pixels. Thus, I am going to use fillPortion, which works a bit like bootstrap where they will designate a specified ratio of the width of each element with the fillPortion.
```

Comment:
Can you break this line of text into smaller ones?

Reply:
Yeah sure, I got it